config/bsb_db.json

```diff
@@ -6972,7 +6981,7 @@
     "Melbourne",
     "VIC",
     "3000",
-    "PE "
+    " EH"
```

Comment:
Should we strip leading space?

Reply:
I'd prefer to avoid modifying the fetched source data and [it seems](https://github.com/zeptofs/bsb/blob/4699030f739a12ce71ff173f3e627168ffaeb491/lib/bsb.rb#L21-L23) that character position matters - they're using the entry value as a 3-character vector of flags.